config/routes.rb

```diff
@@ -35,6 +35,9 @@
     get 'agree_to_terms_of_deposit', as: 'agree_to_terms_of_deposit', on: :collection
   end
 
+  resources :contacts
```

Comment:
Could we do `resource :contact` here (note the singular).  Then we don't need the alias below, right?

Reply:
http://guides.rubyonrails.org/routing.html#singular-resources